test/unit/range/container/dynamic_bitset_test.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <gtest/gtest.h>
 
+#include <seqan3/core/bit_manipulation.hpp>
```

Comment:
What was missing?

Reply:
The test uses `EXPECT_EQ(seqan3::detail::sizeof_bits<decltype(*std::declval<seqan3::dynamic_bitset<58>>().raw_data())>, 64u);`